### PR TITLE
Refactor public event page to enable speakers call

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

Enables the speakers call which were previously disabled
#### Changes proposed in this pull request:

- Enable speakers call
- remove the snake case variables in index router
image

### CODE REVIEW

File: app/serializers/event.js [hunk 1]
```diff
@@ -2,13 +2,14 @@ import ApplicationSerializer from 'open-event-frontend/serializers/application';
...
-  attrs: {
+  primaryKey : 'attributes.identifier',
+  attrs      : {
     type             : 'event-type',
     topic            : 'event-topic',
     subTopic         : 'event-sub-topic',
     copyright        : 'event-copyright',
-    externalEventUrl : 'event-url'
+    externalEventUrl : 'event-url',
+    speakersCall     : 'speakers-call'
```
Comment (niranjan94) [inline app/serializers/event.js]: Why is this needed?

Comment (CosmicCoder96) [inline app/serializers/event.js]: Sorry , removed that. It isn't required no event prefix 😅

Comment (niranjan94) [inline app/serializers/event.js]: Also @CosmicCoder96 could you also remove `externalEventUrl : 'event-url',` not needed anymore. The API response attribute name has changed.

Comment (CosmicCoder96) [inline app/serializers/event.js]: @niranjan94 done !

Comment (niranjan94) [inline app/serializers/event.js]: Thank you :)
